server/storage/players.lua

```diff
@@ -128,6 +128,69 @@ local function fetchPlayerEntity(citizenId)
     } or nil
 end
 
+---@param filters table<string, any>
+local function handleSearchFilters(filters)
+    if not (filters) then return '', {} end
+    local holders = {}
+    local clauses = {}
+    if filters.license then
+        clauses[#clauses + 1] = 'license = ?'
+        holders[#holders + 1] = filters.license
+    end
+    if filters.job then
+        clauses[#clauses + 1] = 'JSON_EXTRACT(job, "$.name") = ?'
+        holders[#holders + 1] = filters.job
+    end
+    if filters.gang then
+        clauses[#clauses + 1] = 'JSON_EXTRACT(gang, "$.name") = ?'
+        holders[#holders + 1] = filters.gang
+    end
+    if filters.metadata then
+        local typeof, value = filters.metadata.typeof, filters.metadata.value
```

Comment:
Could the type be derived from the value instead of passed?

Reply:
Could you elaborate on this one?
As typeof is the metadata key to fetch like "ishandcuffed" "inlaststand" etc
value is just the value the user is looking for like "true" "false" "1" etc

Reply:
Oh. In that case wouldn't it make more sense to use a key value table for metadata rather than two named keys (typeof, and value)?

Reply:
Done as requested